src/helpers.php

```diff
@@ -75,7 +75,7 @@ function array_get_list($array, $path)
  *
  * @return boolean
  */
-function is_associative($array)
+function is_associative(array $array): bool
```

Comment:
let's remove this helpers as it already implemented in the min Laravel version https://laravel.com/docs/10.x/helpers#method-array-isassoc

Reply:
Done